### PR TITLE
Prometheus: Move oss-fuzz downstream

### PR DESCRIPTION
**Pull request by roidelapluie**
Pending https://github.com/prometheus/prometheus/pull/8464

Signed-off-by: Julien Pivotto <roidelapluie@inuits.eu>

### CODE REVIEW

File: projects/prometheus/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,6 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN go get github.com/prometheus/prometheus/cmd/...
```
Comment (inferno-chromium) [inline projects/prometheus/Dockerfile:18]: You forgot to add git clone to your repo :)

**Review by inferno-chromium — CHANGES_REQUESTED** [review]
Please verify your changes locally using 
https://google.github.io/oss-fuzz/advanced-topics/reproducing/#reproducing-build-failures

**Comment by roidelapluie** [comment]
I am wondering how I can either clone master or clone the pull request being built (fir cifuzz)

**Comment by inferno-chromium** [comment]
> I am wondering how I can either clone master or clone the pull request being built (for cifuzz)

cifuzz uses github actions, so it auto uses the pull request code, no need to clone, just need to clone this for oss-fuzz in Dockerfile. see https://google.github.io/oss-fuzz/getting-started/continuous-integration/

**Pull request closed without merging**